scikits/umfpack/tests/test_interface.py

```diff
@@ -62,6 +64,19 @@ def test_splu_solve(self):
         x2 = lu.solve(self.b2)
         assert_allclose(a*x2, self.b2)
 
+    def test_splu_solve_sparse(self):
+        # Prefactorize (with UMFPACK) matrix for solving with multiple rhs
+        A = self.a.astype('d')
+        lu = um.splu(A)
+
+        b = csc_matrix(self.b.reshape(self.b.shape[0], 1))
+        b2 = csc_matrix(self.b2.reshape(self.b2.shape[0], 1))
+        B = hstack((b, b2))
+
+        X = lu.solve_sparse(B)
+        assert dense_norm(((A*X) - B).todense()) < 1e-10
```

Comment:
I have those two entries with values: -1.77635684e-15, 8.88178420e-16 - this is numerically zero (machine epsilon is 2.220446049250313e-16). IMHO you can test for 1e-14 instead of 1e-10.